packages/api/pages/explorer/index.tsx

```diff
@@ -0,0 +1,8 @@
+import dynamic from 'next/dynamic';
+import 'swagger-ui-react/swagger-ui.css';
+
+const SwaggerUI = dynamic(() => import('swagger-ui-react'), { ssr: false });
+
+export default function ApiDocsView() {
+  return <SwaggerUI url="/spec/index.yml" />;
```

Comment:
This UI belongs on the server side because it is docs for that API. That also prevents normal users from needing to download these resources when they will never see them in the app.